src/webhook_service/terraform/lambda.tf

```diff
@@ -159,6 +157,135 @@ module "get_webhook_docker_image" {
   }
 }
 
+####################################
+####################################
+####################################
+# GET /webhooks ####################
+####################################
+####################################
+####################################
+
+module "list_webhooks_lambda" {
+  source  = "terraform-aws-modules/lambda/aws"
+  version = "7.7.0"
+
+  function_name  = local.list_webhooks_function_name_and_ecr_repo_name
+  description    = "AWS Educate TPET ${var.service_hyphen} in ${var.environment}: GET /webhooks"
+  create_package = false
+  timeout        = 30
+
+  ##################
+  # Container Image
+  ##################
+  package_type = "Image"
+  # architectures = ["x86_64"] # or ["arm64"]
+  architectures = ["arm64"]
```

Comment:
```suggestion
  architectures = ["x86_64"]
```

Reply:
Hi @sh1un, I was wondering if i am using macos and does is matter to build x86_64 image here?Thanks

Reply:
Hi @chungchihhan,

The Free plan of GitHub Actions does not provide an arm64 runner. This is why we need to set the architecture to `x86_64`. However, during development, it is fine to set it to `arm64`. Just make sure not to push this code change to the repository.

Reply:
The pushed code on [origin](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/terraform/lambda.tf) is all set with the `x86_64` image architecture.